### PR TITLE
Feature/DDBP-2456 - Add client name and Case ref to submitted email

### PR DESCRIPTION
**Pull request by acsauk-mojd**
I've had an initial go at drying some of the translation files out where it made sense as there was a lot of repeated markup - not sure if theres a better way to achieve this?

### CODE REVIEW

File: src/AppBundle/Service/Mailer/MailFactory.php [hunk 1]
```diff
@@ -259,8 +259,9 @@ public function createReportSubmissionConfirmationEmail(EntityDir\User $user, En
         $viewParams = [
             'submittedReport' => $submittedReport,
             'newReport'       => $newReport,
-            'deputyFirstName' => $user->getFirstname() . ' ' . $user->getLastname(),
-            'newReport'       => $newReport,
```
Comment (slizzio) [inline src/AppBundle/Service/Mailer/MailFactory.php]: why was this removed?

File: src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig [hunk 2]
```diff
@@ -4,18 +4,26 @@
...
+    '%homepageUrl%': homepageUrl,
+    '%fullClientName%': fullClientName,
+    '%caseNumber%': caseNumber
 } %}
 
 {% block emailHTMLHeaderTitle %}{{ ('default.htmlHeaderTitle') | trans }}{% endblock %}
 
 {% block content %}
 
+    {% if recipientRole != 'default' %}
```
Comment (slizzio) [inline src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig]: Professionals and PAs only?

Comment (acsauk-mojd) [reply src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig]: Yeah - confirmed in the ticket. From what I could see in `MailFactory::getRecipientRole()`, anything other than PA or Professional will be set to `default` so hopefully this should be enough?

Comment (slizzio) [reply src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig]: Yes, checked and confirmed, default it seems to be.

Comment (slizzio) [reply src/AppBundle/Resources/views/Email/report-submission-confirm.html.twig]: Confirmed

**Comment by acsauk-mojd** [comment]
Closing in favour of https://github.com/ministryofjustice/opg-digi-deps-client/pull/1163 as jenkins cant build from branches that contain a `/`.
